### PR TITLE
Checking for composer autoloader

### PR DESCRIPTION
**Pull request by WyriHaximus**
AS my previous PR (#18) was borked I've made a new one, with a little more shiny in it. It now checks for an ENV variable for the correct composer vendor directory and then check if the file exists and adds it to bootstrap.php

### CODE REVIEW

**Comment by dereuromark** [comment]
Yeah would have been nice to include this one!

File: README.markdown [hunk 1]
```diff
@@ -77,6 +77,10 @@ Should you wish to manually setup a plugin, templates for the following files ar
 
 Using the `setup.sh` script as described above will use these files as templates.
 
+## Composer
+
+`before_scripts.sh` will check for `composer.json` in your plugin and load all dependencies if it finds one. It also adds loading of the autoloader to `bootstrap.php` and uses the `COMPOSER_VENDOR` environment variable for that. By default it is set to `Vendor`, so if yours is different, `vendor` for example, be sure to update `.travis.yml` accordingly. 
+
```
Comment (dereuromark) [inline README.markdown]: isnt vendor and Vendor the other way around, looking at your code below.

File: templates/.travis.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ php:
 env:
   global:
     - PLUGIN_NAME=Example
+    - COMPOSER_VENDOR=Vendor
```
Comment (dereuromark) [inline templates/.travis.yml]: the text above `By default it is set to `Vendor`` would not be true otherwise.

**Comment by WyriHaximus** [comment]
@dereuromark my rebase-foo is weak but I'll have a got at it :). Thanks for the ping @jadb completely missed this message.

**Comment by dereuromark** [comment]
@WyriHaximus Is this PR still valid?

**Comment by WyriHaximus** [comment]
@dereuromark honestly I completely forgot about this. Also since cake 3, it's trivial to setup travis for plugins so I personally don't have a need for it anymore. So I would say this PR has some come irrelevant and invalid unless anyone thinks otherwise.

**Comment by dereuromark** [comment]
For the 2.x plugins maybe.

**Comment by WyriHaximus** [comment]
Maybe, but I wouldn't be the right person to ask that. If anyone thinks this is still useful then go ahead lets finish it but my cake 2.x is rather rusty

**Comment by dereuromark** [comment]
So whats the status here?

**Comment by WyriHaximus** [comment]
Abandoned

**Pull request closed without merging**